packages/webpack-stats-differ/package.json

```diff
@@ -20,20 +20,24 @@
     "lib/**/*.d.ts"
   ],
   "dependencies": {
-    "matcher": "^3.0.0",
-    "multimatch": "^5.0.0",
+    "@microsoft/azure-artifact-storage": "*",
+    "@mixer/webpack-bundle-compare": "^0.1.0",
     "globby": "^11.0.2",
     "lodash": "^4.17.21",
-    "upath": "^2.0.1",
-    "@mixer/webpack-bundle-compare": "^0.1.0",
-    "@microsoft/azure-artifact-storage": "*"
+    "matcher": "^3.0.0",
+    "multimatch": "^5.0.0",
+    "stream-chain": "^2.2.5",
+    "stream-json": "^1.7.5",
+    "upath": "^2.0.1"
```

Comment:
Where do we use upath ?

Reply:
My bad. Removed it.

Reply:
I thought I accidentally added it when working on this but seems it was already there and used here https://github.com/microsoft/ardiffact/blob/main/packages/webpack-stats-differ/src/getFriendlyAssetName.ts

Reply:
I thought I accidentally added it when working on this but seems it was already there and used here https://github.com/microsoft/ardiffact/blob/main/packages/webpack-stats-differ/src/getFriendlyAssetName.ts